DigitalLearningSolutions.Web/Views/SuperAdmin/Users/Index.cshtml

```diff
@@ -1,24 +1,117 @@
-    <nav class="side-nav-menu" aria-label="Side navigation bar">
-      <partial name="~/Views/SuperAdmin/Shared/_UserAccountsSideNavMenu.cshtml" model="Model.CurrentPage" />
-    </nav>
-  </div>
+    <div class="nhsuk-grid-column-full">
+        <div id="no-js-styling">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-one-quarter">
+                    <nav class="side-nav-menu" aria-label="Side navigation bar">
+                        <partial name="~/Views/SuperAdmin/Shared/_UserAccountsSideNavMenu.cshtml" model="Model.CurrentPage" />
+                    </nav>
+                </div>
 
-  <div class="nhsuk-grid-column-three-quarters">
-    <div class="nhsuk-grid-row">
-      <h1 class="nhsuk-heading-xl">User Accounts</h1>
-    </div>
+                <div class="nhsuk-grid-column-three-quarters">
+                    <div class="nhsuk-u-margin-bottom-4">
+                        <h1 class="nhsuk-heading-xl">@ViewData["Title"]</h1>
+                    </div>
+                    <form class="nhsuk-u-margin-bottom-3" method="get" role="search" asp-controller="Users" asp-action="Index" asp-route-page="@Model.Page">
+                        <div class="nhsuk-grid-row">
+                            <div id="search">
+                                <div class="nhsuk-grid-column-one-quarter">
+                                    <vc:numeric-input asp-for="@nameof(Model.UserId)"
+                                                      label="User ID"
+                                                      populate-with-current-value="true"
+                                                      type="number"
+                                                      hint-text=""
+                                                      css-class=""
+                                                      required="true" />
+                                </div>
+                                <div class="nhsuk-grid-column-one-half">
+                                    <vc:text-input asp-for="@nameof(Model.Search)"
+                                                   label="Search"
+                                                   populate-with-current-value="true"
+                                                   type="text"
+                                                   spell-check="false"
+                                                   hint-text=""
+                                                   autocomplete=""
+                                                   css-class=""
+                                                   required="true"
+                                                   placeholder-text="Search name, email, PRN" />
+                                </div>
+                                <div class="nhsuk-grid-column-one-quarter">
+                                    <button class="nhsuk-button items-per-page-submit nhsuk-u-margin-top-5" type="submit">Search</button>
```

Comment:
This would be better as a standard button since it's detached from the search input, I think. Should just be a case of losing the "items-per-page-submit" class.